src/main/java/frc/robot/commands/ArmJoystickConeCommand.java

```diff
@@ -102,7 +102,7 @@ public void execute() {
       double z = m_joystick.getRawAxis(XboxController.Axis.kRightY.value);
       z = MathUtil.applyDeadband(z, ArmConfig.ARM_JOYSTICK_DEADBAND);
       z_offset += z * -0.16;
-      z_offset = MathUtil.clamp(z_offset, -8, 0);
+      z_offset = MathUtil.clamp(z_offset, -8, 0); // maybe change higher limit to -1?
```

Comment:
Changing the higher limit to -1 would cause cone middle and cone top setpoints to always be 1 inch below what it should be.